gradle-plugin/build.gradle

```diff
@@ -18,6 +18,8 @@ plugins {
   id 'com.gradle.plugin-publish' version "0.12.0"
   id 'groovy'
   id 'java-gradle-plugin'
+  id 'maven-publish'
+  id 'net.researchgate.release' version '2.8.1'
```

Comment:
Jib also uses this plugin https://github.com/GoogleContainerTools/jib/blob/master/jib-gradle-plugin/build.gradle#L3